nixos/modules/security/acme.nix

```diff
@@ -248,13 +272,18 @@ let
 
       # Working directory will be /tmp
       script = ''
-        set -euo pipefail
+        set -euxo pipefail
```

Comment:
Oh man, is this annoying.

Reply:
How so? I added it because debugging people's issues without the shell commands in the logs is quite difficult. Also so much functionality of cert renewal happens in bash. Personally I've found it quite useful.

Reply:
But shouldn't debugging output only be enable when you're, well, debugging?

Reply:
Well with acme, you know you want to debug only after the fact. besides, the amount of logs this generates is not that big...